app/assets/javascripts/add_date_field.js.coffee

```diff
@@ -164,4 +176,15 @@
 
       date_default()
       date_range(selected_date)
+
+    $(document).on("managed_field:add", '.multi_value_date.form-group', (event) ->
+      window.setTimeout(->
+        date_multiple_default()
```

Comment:
Does `event.target` refer to the new managed field that was added? And if so, it would be a good idea to pass that to `date_multiple_default()` and call .datetimepicker directly on it instead of the current broad jquery selector for all of the fields that would have already had the .datetimepicker called on them.